client/my-sites/plan-features-comparison/index.jsx

```diff
@@ -313,22 +309,6 @@ PlanFeaturesComparison.defaultProps = {
 	busyOnUpgradeClick: false,
 };
 
-export const calculatePlanCredits = ( state, siteId, planProperties ) =>
-	planProperties
-		.map( ( { planName, availableForPurchase } ) => {
-			if ( ! availableForPurchase ) {
-				return 0;
-			}
-			const annualDiscountPrice = getPlanDiscountedRawPrice( state, siteId, planName );
-			const annualRawPrice = getSitePlanRawPrice( state, siteId, planName );
-			if ( typeof annualDiscountPrice !== 'number' || typeof annualRawPrice !== 'number' ) {
-				return 0;
-			}
-
-			return annualRawPrice - annualDiscountPrice;
-		} )
-		.reduce( ( max, credits ) => Math.max( max, credits ), 0 );
-
```

Comment:
This is essentially a calculation of state which [belongs in a selector](https://redux.js.org/usage/deriving-data-selectors#calculating-derived-data-with-selectors).  And has been moved to   [calculate-plan-upgrade-credits.ts](https://github.com/Automattic/wp-calypso/pull/76100/files#diff-9541c9db66f4bb4bf6587c41379862986af7ceed675f394f868503b0393b0a31)